docs/src/userguide/glossary.rst

```diff
@@ -0,0 +1,194 @@
+        |
+
+   Lazy Data
+        Data stored in hard drive, and then temporarily loaded into RAM in
+        batches when needed. Allows of less memory usage and faster performance,
+        thanks to parallel processing.
+
+        | **Related:** :term:`Dask` **|** :term:`Real Data`
+        | **More information:** :doc:`real_and_lazy_data`
+        |
+
+   Long Name
+        A name describing a :term:`phenomenon`, not limited to the
+        the same restraints as :term:`standard name`.
+
+        | **Related:** :term:`Standard Name` **|** :term:`Cubes`
+        | **More information:** :doc:`iris_cubes`
+        |
+
+   Matplotlib
+        A python package for plotting and projecting data in a wide variety
+        of formats.
+
+        | **Related:** :term:`CartoPy` **|** :term:`NumPy`
+        | **More information:** `Matplotlib <https://scitools.org.uk/cartopy/docs/latest/>`_
+        |
+
+   Metadata
+        The data which is used to describe phenomenon data e.g. longitude.
+
+        | **Related:** :term:`Phenomenon` **|** :term:`Cubes`
+        | **More information:** :doc:`../further_topics/metadata`
+        |
+
+   NetCDF Format
+        A meteorological file format; this is the data model
+        iris is based on. Follows `CF Conventions <http://cfconventions.org/>`_.
+
+        | **Related:** :term:`Fields File (FF) Format`
+         **|** :term:`GRIB Format` **|** :term:`Post Processing (PP) Format`
+        | **More information:** `NetCD-4 Python Git <https://github.com/Unidata/netcdf4-python>`_
+        |
+
+   NumPy
+        A mathematical Python library, predominantly based around
+        multi-dimensional arrays.
+
+        | **Related:** :term:`Dask`  **|** :term:`Cubes`
+         **|** :term:`Xarray`
+        | **More information:** `NumPy.org <https://numpy.org/>`_
+        |
+
+   Phenomenon
+        The primary data which is measured, usually within a cube, e.g.
+        air temperature.
+
+        | **Related:** :term:`Metadata` **|** :term:`Cubes`
+        | **More information:** :doc:`iris_cubes`
+        |
+
+   Post Processing (PP) Format
+        A meteorological file format, created from a post processed
+        :term:`Fields File (FF) Format`.
+
+        | **Related:** :term:`GRIB Format` **|** :term:`NetCDF Format`
+        | **More information:** `PP Wikipedia Page <https://en.wikipedia.org/wiki/PP-format>`_
+        |
+
+   Real Data
+        Data that has been loaded into RAM, as opposed to sitting
+        on the hard drive.
+
+        | **Related:** :term:`Lazy Data`
+        | **More information:** :doc:`real_and_lazy_data`
+        |
+
+   Standard Name
+        A name describing a :term:`phenomenon`, keeping within
+        bounds of `CF Standardisation <http://cfconventions.org/standard-names.html>`_.
+
+        | **Related:** :term:`Long Name` **|** :term:`Cubes`
+        | **More information:** :doc:`iris_cubes`
+        |
+
+   Units
+        The unit with which the phenomenon is measured.
```

Comment:
```suggestion
        The physical unit of measure of a :term:`phenomenon`, e.g. "m / sec".
```

Reply:
Slightly edited this one; the example and link are useful, but I'm not sure the definition itself adds much, apart from removing some colloquialism, which is something I was striving for with this, and frankly still don't feel I've reached.